panther_manager/src/plugins/action/call_set_bool_service_node.cpp

```diff
@@ -0,0 +1,45 @@
+// Copyright 2024 Husarion sp. z o.o.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <panther_manager/plugins/action/call_set_bool_service_node.hpp>
+namespace panther_manager
+{
+
+bool CallSetBoolService::setRequest(typename Request::SharedPtr & request)
+{
+  if (!getInput<bool>("data", request->data)) {
+    RCLCPP_ERROR_STREAM(node_->get_logger(), "Failed to get input [data]");
+    return false;
+  }
+  return true;
+}
+
+BT::NodeStatus CallSetBoolService::onResponseReceived(const typename Response::SharedPtr & response)
+{
+  if (!response->success) {
+    RCLCPP_ERROR_STREAM(
+      node_->get_logger(),
```

Comment:
```suggestion
      this->node_->get_logger(),
```
applies to other usage of protected `node_` object

Reply:
I am opposed to add `this` but I see we use `this` earlier in the `gpiod` and the `battery`. I will add but in this case we should add the `this` guide to the code style guide https://docs.google.com/document/d/1KjSnz3e6u0Vuw17-5XoXsnaJjgpyR-Fo0F4s5Nu3WPE/edit.